Scripts/dea_spatialtools.py

```diff
@@ -419,9 +419,16 @@ def contours_to_multiline(da_i, z_value, min_vertices=2):
         
         # Extracts contours from array, and converts each discrete
         # contour into a Shapely LineString feature
-        line_features = [LineString(i[:,[1, 0]]) 
-                         for i in find_contours(da_i.data, z_value)
-                         if i.shape[0] > min_vertices]
+        try:
+            line_features = [LineString(i[:,[1, 0]]) 
+                             for i in find_contours(da_i.data, z_value) 
+                             if i.shape[0] > min_vertices]
+            
+        except:
```

Comment:
Naked exceptions aren't great - please choose a specific exception! It'll also be easier to read - I'm not sure what you're trying to catch here.